### ISSUE TITLE
Remember Exif Panel width always

### ISSUE THREAD

**Issue by stefonarch** [lxqt/lximage-qt#463]
Opening LXimage-qt the exiv bar is small and has to be always manually enlarged to access the info.
Closing and reopening the exiv bar size is remembered, closing the app not.
##### Expected Behavior

The exiv bar opens always in a usable size
##### Current Behavior

Customization is lost on close.
##### Possible Solution

Not sure, a minimal default size, but could conflict with thumbnails preview on the right, which has fixed width.
##### Steps to Reproduce (for bugs)

1. Open an image in LXimage-qt in fullscreen
2. "Show exif data" in menu > view.
3. to see the info size has to be changed, close app
4. Reopen, size is small again.

##### Context

I'm using often exif information, configuring a shortcut (E) is convenient, but every time adjusting the size is less convenient.
##### System Information

 Version: Git master

**Comment by tsujan** [comment]
> but could conflict with thumbnails preview on the right

Yes. I wanted to ask the contributor to exclude the right side with the thumbnail-bar but I didn't have time for a detailed explanation; so, I just ignored it.

**Comment by stefonarch** [comment]
Btw: thumbnails positions left, right, bottom or top aren't translatable.
E

**Comment by tsujan** [comment]
Will look into them.
